code/style.css

```diff
@@ -35,24 +88,133 @@ p {
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  border: 2px solid black;
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
   background: midnightblue;
-
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  border: 2px solid black;
 }
 
-button {
-  padding: 8px 16px;
+.button-reveal {
+  padding: 5px 16px;
   background: crimson;
-  border: none;
+  border: 1px solid black;
   font-size: 20px;
   border-radius: 8px;
   display: block;
-  margin: 0 auto;
+  margin: 2px auto;
+  font-family: 'Architects Daughter', cursive;
+  font-weight: bold;
+}
+
+.button-reset {
+  padding: 5px 16px;
+  background: crimson;
+  border: 1px solid black;
+  font-size: 16px;
+  border-radius: 8px;
+ 
+  margin: 2px auto;
+  font-family: 'Architects Daughter', cursive;
+  font-weight: bold;
 }
+
+.button-reveal:hover {
+  opacity: 0.9;
+  cursor: pointer;
+}
+
+.button-reset:hover {
+  opacity: 0.9;
+  cursor: pointer;
+}
+
+.weapon-img-div, .room-img-div {
+  background-color: #fff;
+  display: inline-block;
+  margin: 25px auto;
+  padding: 15px 0;
+  border-radius: 50%;
+}
+
+.killer-image {
+  border-radius: 50%;
+  margin: 25px 0;
+  height: 150px;
+}
+
+.weapon-image, .room-image {
+  height: 110px;
+}
+
+.reveal-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 50%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.reveal-section h2 {
+  background: rgba(204, 204, 204, 0.9);
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.loader {
+  background: rgb(250, 246, 246);
+  border-radius: 50%;
+  animation-duration: 3s;
+  animation-iteration-count: infinite;
+  animation-name: bounce;
+  animation-timing-function: alternate;
+  visibility: hidden;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+@keyframes bounce {
+  0% {
+    width: 16px;
+    height: 16px;
```

Comment:
These properties never change, no need to add them in `@keyframes`. The best approach would be to add them to `.loader` class